wpilibj/src/main/java/edu/wpi/first/wpilibj/DoubleSolenoid.java

```diff
@@ -70,10 +70,6 @@ public DoubleSolenoid(PneumaticsBase module, final int forwardChannel, final int
       }
     }
 
-    if (module.checkAndReserveSolenoids(m_mask) != 0) {
```

Comment:
This is definitely not something to delete. Necessary for proper functionality

Reply:
Oh nevermind. Ignore me I didn't realize we had already done it above.

Reply:
Unit test FTW 😉